src/ensembl/src/content/app/entity-viewer/gene-view/components/transcripts-table/TranscriptsTable.scss

```diff
@@ -0,0 +1,93 @@
+@import 'src/styles/common';
+
+.table {
+  display: grid;
+  grid-template-columns: 1fr 3fr 1fr 1fr 2fr;
+  grid-template-rows: auto;
+  min-width: 1900px;
+}
+
+.tableRow {
+  border-bottom: 1px solid $dark-grey;
+}
+
+.tableCell {
+  display: inline-block;
+  padding: 8px 4px;
+}
+
+.tableBody {
+  padding-top: 20px;
+}
+
+.tableHeader {
+  color: $grey;
+
+  .tableRow {
+    border-bottom: 1px solid #8d621c;
```

Comment:
What color is this? Is it not our regular orange color? If not, is it a recurrent color that should be added to the color palette?

Reply:
It should be the same orange that's used in the transcript scale

Reply:
Weird, I was using the `orange` variable from settings.scss for them, but that's not `#8d621c`.

Reply:
:heavy_check_mark: